lib/api/kuzzle.js

```diff
@@ -213,7 +232,7 @@ function registerErrorHandlers(kuzzle) {
       });
   });
 
-  Object.keys(coreDumpSigals).forEach((signal) => {
+  Object.keys(coreDumpSigals).forEach(signal => {
     process.removeAllListeners(signal);
     process.on(signal, () => {
       console.error(`ERROR: Caught signal: ${signal}`); // eslint-disable-line no-console
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 89 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ATrailingComment)